src/core_plugins/metric_vis/public/metric_vis_params.html

```diff
@@ -195,7 +248,12 @@
 
         <div class="kuiSideBarFormRow">
           <label class="kuiSideBarFormRow__label" for="metricFontSize">
-            Font Size (<span ng-bind="editorState.params.metric.style.fontSize"></span>pt)
+            <span
```

Comment:
This should be a single i18n string, instead of splitting it up. The current way we assume, that you can use `{Label} ({Size}pt)` in every language, which imho won't hold true. Not every language might want to use Latin parenthesis, not every language might want to have that order, etc. I suggest we put this into a single i18n string, just passing in the font size as a paramter.